src/bootstrap/bin/rustc.rs

```diff
@@ -125,6 +124,17 @@ fn main() {
         cmd.arg(format!("-Cdebuginfo={}", debuginfo_level));
     }
 
+    if env::var_os("RUSTC_DENY_WARNINGS").is_some() &&
+       env::var_os("RUSTC_EXTERNAL_TOOL").is_none() {
+        cmd.arg("-Dwarnings");
+        cmd.arg("-Drust_2018_idioms");
```

Comment:
It looks like we maybe lost bare_trait_objects here? Or is that included in rust_2018_idioms (it should be, I'd guess)?

Reply:
Yes, it's a part of the edition idiom group.
(And it's a part of `-Dwarnings` too now because it was turned into warn-by-default on both editions.)